### ISSUE TITLE
[hdfwriter] Document I3SimHDFWriter (Trac #2229)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2221]
New users are confused by Q frames, and it really doesn't help that the simple solution, swapping I3HDFWriter to I3SimHDFWriter, appears to be completely undocumented. See: 
https://icecube-spno.slack.com/archives/C02KQL9KN/p1547334442211400

Migrated from https://code.icecube.wisc.edu/ticket/2229
```json
{
    "status": "closed", 
    "changetime": "2019-02-14T15:23:42", 
    "description": "New users are confused by Q frames, and it really doesn't help that the simple solution, swapping I3HDFWriter to I3SimHDFWriter, appears to be completely undocumented. See: \nhttps://icecube-spno.slack.com/archives/C02KQL9KN/p1547334442211400", 
    "reporter": "jvansanten", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550157822842486", 
    "component": "combo core", 
    "summary": "[hdfwriter] Document I3SimHDFWriter", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2019-01-13T13:14:29", 
    "milestone": "Vernal Equinox 2019", 
    "owner": "jvansanten", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:52:56: olivas changed milestone from "" to "Vernal Equinox 2019"

**Comment by icecube-trac** [comment]
Trac update at 20190214T15:23:42: 
* jvansanten commented:

> In changeset 171291:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="171291"
> Document I3SimHDFWriter. Fixes #2221
> ```


* jvansanten changed status from "new" to "closed"
* jvansanten changed resolution from "" to "fixed"
